packages/react-app-revamp/components/_pages/WizardFormCreateContest/Step3/useDeployContest.ts

```diff
@@ -77,12 +77,14 @@ export function useDeployContest(form: any) {
         // downvoting: is downvoting allowed in this contest
         // 0 = false; 1 = true
         values?.downvotingAllowed === true ? 1 : 0,
+        values?.useSameTokenForSubmissions === true ? 0 : 1,
```

Comment:
Shouldn't this be `values?.useSameTokenForSubmissions === true ? 1 : 0,` because we want 1 if `true` and 0 if `false`?

Reply:
Ah my bad, read this from here 
```
  await deploy("Contest", {
    // Learn more about args here: https://www.npmjs.com/package/hardhat-deploy#deploymentsdeploy
    from: deployer,
    args: ["joking", "Enter a joke", GenericVotesTimestampToken.address, GenericVotesTimestampToken.address, [5, 10, 10, 7, ethers.utils.parseEther("1"), 2, 100, 1, 0]],
    log: true,
    waitConfirmations: 5,
  });

```
Since we use `GenericVotesTimestampToken.address` for voting and submissions here, I thought `_initialSubmissionGatingByVotingToken` was `0` when both submission and voting tokens are the same :thinking:

Reply:
Ahh yeah I can see how that is confusing, I'm sorry about that! If you look in `packages/hardhat/contracts/governance/Governor.sol` in the files changed for this PR `submissionGatingByVotingToken()` should be `1` if the voting and submission tokens are the same and `0` if they are not.

Reply:
Just updated the deploy script to be less confusing.